internal/gatewayapi/runner/runner.go

```diff
@@ -96,7 +98,11 @@ func (r *Runner) subscribeAndTranslate(ctx context.Context) {
 					t.ExtensionGroupKinds = extGKs
 				}
 				// Translate to IR
-				result := t.Translate(resources)
+				result, err := t.Translate(resources)
+				if err != nil {
+					// Currently all errors that Translate returns should just be logged
+					r.Logger.Error(err, "errors detected during translation")
```

Comment:
non blocking comment, something we can add in a follow up
is to send the err to the `errChan` similar to
https://github.com/envoyproxy/gateway/blob/3772293d83239bc568e62d3feac7f46e91c3cb6d/internal/infrastructure/runner/runner.go#L84C6-L84C20
to bump the metrics